ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_rounding.py

```diff
@@ -58,3 +58,35 @@ def test_numpy_ceil(
         subok=True,
         test_values=False,
     )
+
+
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=ivy_np.valid_float_dtypes,
+    ),
+    as_variable=helpers.array_bools(num_arrays=1),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.numpy.fix"
+    ),
+    native_array=helpers.array_bools(num_arrays=1),
```

Comment:
similarly `native_array` is not required to be initialized

Reply:
Done